### DISCUSSION TITLE
error: either backend url or subdomain must be set

### DISCUSSION THREAD

**Discussion by ankur1163** [nhost/nhost#1463] [Q&A] [answered]
I am trying to work with nhost in react native. I am getting this error. I am following docs and one youtube video. 
1) In video, i have to wrap my components around <NhostProvider nhost={nhost}> , but i think in docs, it is not mentioned. Do i have to wrap around it or not? 
2) error: either backend url or subdomain must be set , i see this error in my android emulator, how do i solve this? 

I have created env.local file. I have provided subdomain in it and then exported it in my App.js 
```
const nhost = new NhostClient({
    subdomain: process.env.REACT_APP_NHOST_SUBDOMAIN,
    region: process.env.REACT_APP_NHOST_REGION,
  });
```

Thanks in advance

**Comment by ankur1163** [comment, marked as answer]
i solved it

**Reply by elitan** [reply]
For the next developer who might have a similar issue, how did you solve it?
